templates/ruby/lib/container/payload.rb.twig

```diff
@@ -0,0 +1,77 @@
+module Appwrite
+  class Payload
+    attr_reader :data
+    attr_reader :filename
+
+    def initialize(data: nil, path: nil, filename: nil)
+      @path = path
+      @data = data
+      @filename = filename
+
+      @size = if @data then
+        @data.bytesize
+      else
+        File.size(@path)
+      end
+    end
+
+    def read(offset: 0, length: nil)
```

Comment:
Having generic read seems off. Would read work with strings or jsons? Is this needed specifically for files?

Reply:
It's need for the chunking logic in the client